tox.ini

```diff
@@ -45,8 +45,7 @@ commands =
 
     ethereum-spec-diff \
         "{toxworkdir}/docs/stage0_out/autoapi/" \
-        "doc/diffs" \
-        --compare-sections-normally
+        "doc/diffs"
```

Comment:
I wouldn't expect this to make a huge difference, since we don't generally change the order of sections. Is this mostly for inserted functions?

Reply:
Yes. This is is to handle the inserted functions properly